lib/ukrandom/dev.c

```diff
@@ -79,13 +70,17 @@ static int dev_random_read(struct device *dev __unused, struct uio *uio,
 {
 	__sz count;
 	char *buf;
+	int ret;
 
 	buf = uio->uio_iov->iov_base;
 	count = uio->uio_iov->iov_len;
 
-	uk_random_fill_buffer(buf, count);
+	ret = uk_random_fill_buffer(buf, count);
+	if (unlikely(ret))
+		return rc;
```

Comment:
```suggestion
		return ret;
```

Otherwise, this gives a build error. If you enable `LIBUKRANDOM_DEVFS`.

Reply:
Fixed